src/main/java/org/osc/manager/ism/entities/PolicyElements.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.manager.ism.entities;
+
+
+
+
+public class PolicyElements {
```

Comment:
With PolicyEntity in place and implementing ManagerPolicyElement you should no longer need PolicyListElement, it should be quick to replace all references to PolicyListElement with PolicyEntity and delete PolicyListElement.  If you are tracking using the *Entities in the SDK interfaces implementation with different user stories you can do that later but I want to make sure you agree that all the *Entities will implement their corresponding *Element interfaces and there wont any other classes implementing those interfaces like Policy or PolicyListElement when the feature is complete. Also for you to consider an entire entity ready, like Policy, it should be: persisted in the db; exposed with those crud apis in the rest; **and used by the plugin on their SDK interface implementation (this will make OSC also use the persistence mechanism);**